### PR TITLE
ci(clint): remove line length warning

### PR DESCRIPTION
**Pull request by dundargoc**
The warning for exceeding character width is more of an annoyance than
any benefit. The only time it seems to trigger is when a line actually
needs to exceed the limit, such as an URL or a long string message.
Furthermore, both uncrustify and clang-format is already very good at
automatically containing the code at a certain width, rendering manual
checks unnecessary.

### CODE REVIEW

File: src/nvim/ex_docmd.c [hunk 1]
```diff
@@ -2740,7 +2740,6 @@ static void undo_cmdmod(const exarg_T *eap, int save_msg_scroll)
 }
 
 
-
```
Comment (mhinz) [inline src/nvim/ex_docmd.c:2743]: If you do this unrelated change, remove 2 lines. ;P

Comment (dundargoc) [reply src/nvim/ex_docmd.c:2743]: fixed

**Review by mhinz — APPROVED** [review]

**Comment by mhinz** [comment]
The commit itself seems fine.

Are there are arguments against removing the line limit check?

**Comment by lewis6991** [comment]
I think some limit is worth having, even if it's large.

**Comment by dundargoc** [comment]
We do have a limit in the form of uncrustify. [This PR](https://github.com/neovim/neovim/pull/18563) will automatically ensure that all code is under 100 characters, except for URLS and strings which we don't want to split up anyway.

**Comment by justinmk** [comment]
Commit message / description for all such PRs can be this standard blurb:

```
uncrustify is the source of truth where possible.
Remove any redundant checks from clint.py.
See also https://github.com/neovim/neovim/pull/18563
```
